dcos/http.py

```diff
@@ -77,6 +83,7 @@ def request(method,
         kwargs['headers'] = {'Accept': 'application/json'}
 
     timeout = timeout or _timeout()
+    ssl_verify = _ssl_verify()
```

Comment:
Hey @ryane, we are trying to make the `dcos` package/module usable outside of the DCOS CLI. For example we would like people to use the package/module write Python scripts against DCOS. Requiring `util.get_config()` to succeed (environment variable DCOS_CONFIG points to a valid `dcos.toml`) is a unnecessary requirements. Can we refactor this to instead pass the value in?

I know that we currently require that with `_timeout()` but I consider that a bug and we will fix that ASAP.